ui/src/views/AutogenView.vue

```diff
@@ -957,7 +957,8 @@ export default {
             if (param.name !== key) {
               continue
             }
-            if (input === undefined || input === null || input === '') {
+            if (input === undefined || input === null || (input === '' &&
+              !['updateStoragePool', 'updateHost'].includes(action.api))) {
```

Comment:
@Pearl1594 we have physical networks tags also, I just checked we could not delete the tag on physical network too, can you please add that here.